clusterloader2/pkg/measurement/common/slos/pod_startup_latency.go

```diff
@@ -264,6 +264,10 @@ func (p *podStartupLatencyMeasurement) gatherScheduleTimes(c clientset.Interface
 	for _, event := range schedEvents.Items {
 		key := createMetaNamespaceKey(event.InvolvedObject.Namespace, event.InvolvedObject.Name)
 		if _, ok := p.createTimes[key]; ok {
+			if !event.EventTime.IsZero() {
+				p.scheduleTimes[key] = (metav1.Time)(event.EventTime)
+				continue
```

Comment:
else {
  p.schedulerTimes...
}

continue is always more dangerous in the future.

Reply:
Done.